torchci/rockset/commons/__sql/flaky_tests_across_jobs.sql

```diff
@@ -0,0 +1,103 @@
+with failed_jobs as (
+    SELECT
+        FIRST_VALUE(job.conclusion) OVER(
+            PARTITION BY CONCAT(w.name, ' / ', job.name)
+            ORDER BY
+                push.head_commit.timestamp ROWS BETWEEN 1 PRECEDING
+                AND 1 FOLLOWING
+        ) = 'success'
+        and NTH_VALUE(job.conclusion, 2) OVER(
+            PARTITION BY CONCAT(w.name, ' / ', job.name)
+            ORDER BY
+                push.head_commit.timestamp ROWS BETWEEN 1 PRECEDING
+                AND 1 FOLLOWING
+        ) = 'failure'
+        and LAST_VALUE(job.conclusion) OVER(
+            PARTITION BY CONCAT(w.name, ' / ', job.name)
+            ORDER BY
+                push.head_commit.timestamp ROWS BETWEEN 1 PRECEDING
```

Comment:
IMO, it sounds like a good idea to have the number of rows before and after parameterized with a default to 1.  In case we need to tweak it later on.  Thoughts?